src/main/java/com/snowflake/kafka/connector/SnowflakeSinkConnectorConfig.java

```diff
@@ -110,7 +110,7 @@ public class SnowflakeSinkConnectorConfig {
 
   // metrics
   public static final String JMX_OPT = "jmx";
-  public static final boolean JMX_OPT_DEFAULT = true;
+  public static final boolean JMX_OPT_DEFAULT = false;
```

Comment:
Should we do it for snowpipe streaming only? otherwise we need to go through a BCR process